conanfile.txt

```diff
@@ -0,0 +1,19 @@
+# conan usage:
```

Comment:
this is optional, for people using modern tools like the conan package manager...